docs/specs/markdown.yml

```diff
@@ -17,22 +17,47 @@ outputs:
     { "content": "<p><em>this</em> <em>is</em> an <em><strong>inline</strong></em> <code>markdown</code> <strong>test</strong></p>" }
   build.manifest:
 ---
-# Yaml header
+# Yaml header title in lower-case
 inputs:
   docfx.yml:
   docs/a.md: |
     ---
-    title: me
+    title: title from yaml header
     ---
+    # Title from H1
```

Comment:
don't modify this test, it covers the cases where there is a title in the header but no title in markdown